uint32 person_id
BodyPartElm3D nose
BodyPartElm3D neck
BodyPartElm3D right_shoulder
BodyPartElm3D right_elbow
BodyPartElm3D right_wrist
BodyPartElm3D left_shoulder
BodyPartElm3D left_elbow
BodyPartElm3D left_wrist
BodyPartElm3D right_hip
BodyPartElm3D right_knee
BodyPartElm3D right_ankle
BodyPartElm3D left_hip
BodyPartElm3D left_knee
BodyPartElm3D left_ankle
BodyPartElm3D right_eye
BodyPartElm3D left_eye
BodyPartElm3D right_ear
BodyPartElm3D left_ear